app.py

```diff
@@ -13,6 +13,7 @@
     'challenge': 'other',
     'riverRacePvP': 'clan',
     'riverRaceDuel': 'clan',
+    'riverRaceDuelColosseum': 'clan',
```

Comment:
I tried to search for a list of the battles but I didn't found anythin

Reply:
I think this wasn't in the list yesterday as we didn't reach the colloseum yet.